OWNERS

```diff
@@ -16,8 +16,8 @@ reviewers:
 - qu1queee
 - SaschaSchwarze0
 - HeavyWombat
-- ImJasonH
```

Comment:
To remove yourself fully, you should remove yourself also from `approvers`, I think.